packages/cli/templates/vue3/src/style.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
 
```

Comment:
```

html {
  overflow: hidden;
}
```

Not sure why it happens, but in the container, the scroll can become visible after the height has changed:

![2023-04-04_14-09-19 (1)](https://user-images.githubusercontent.com/14206504/229787170-afc96424-4734-4605-bf66-bf5d66920683.gif)